File: python_programming/practices/financial_calculator.py
# MA 6th fincancial calculator

print("What is your monthly income:")
monthly_income = input()
print("what is your monthly rent?")
monthlyRent = input()
print("What is your monthly utilities:")
monthlyUtilities = input()
print("What is your monthly groceries:")
monthlyGroceries = input()
print("What is your monthly transportation:")
monthlyTransportation = input()

rentPercent = int(monthlyRent)/int(monthly_income) * 100
print("Your rent is $", monthlyRent, "and that is", str(rentPercent), "% of your income.")

utilitiesPercent = int(monthlyUtilities)/int(monthly_income) * 100
print("Your ulilities is $", monthlyUtilities, "and that is", str(utilitiesPercent), "% of your income.")

groceriePercent = int(monthlyGroceries)/int(monthly_income) * 100
print("Your groceries are $", monthlyGroceries, "and that is", str(groceriePercent), "% of your income.")

transportationPercent = int(monthlyTransportation)/int(monthly_income) * 100
print("your transportation is $", monthlyTransportation, "and that is", str(transportationPercent), "% of your income.")

savings = int(monthly_income) * 0.1
print("you should save $", str(savings), "a month, that is 10% of you income.")

spendingMoney = int(monthly_income) - (int(monthlyRent) + int(monthlyUtilities) + int(monthlyGroceries) + int(monthlyTransportation) + savings)
print("you have $", str(spendingMoney), "of spending money each month!")

File: c_programming/projects/financial_calculator.c
// MA 6th financial calculator for c
#include <stdio.h>

int main() {
    float income, rent, utilities, groceries, transport;
    float total_expenses, savings, spending;
    
    // Get user input
    printf("What is your monthly income: ");
    scanf("%f", &income);
    
    printf("\nWhat is your monthly rent/mortgage: ");
    scanf("%f", &rent);
    
    printf("What is your monthly utilities: ");
    scanf("%f", &utilities);
    
    printf("What is your monthly groceries: ");
    scanf("%f", &groceries);
    
    printf("What is your monthly transportation: ");
    scanf("%f", &transport);
    
    // Calculate total expenses
    total_expenses = rent + utilities + groceries + transport;
    
    // Save 10% of income
    savings = income * 0.10;
    
    // Calculate remaining spending money
    spending = income - total_expenses - savings;
    
    // Print breakdown with percentages
    printf("\nYour rent is $%.2f and that is %.0f%% of your income.\n", rent, (rent / income) * 100);
    printf("Your utilities are $%.2f and that is %.0f%% of your income.\n", utilities, (utilities / income) * 100);
    printf("Your groceries are $%.2f and that is %.0f%% of your income.\n", groceries, (groceries / income) * 100);
    printf("Your transportation is $%.2f and that is %.0f%% of your income.\n", transport, (transport / income) * 100);
    
    printf("\nYou should save $%.2f a month, that is 10%% of your income.\n", savings);
    printf("You have $%.2f of spending money each month!\n", spending);
    
    return 0;
}
